bullet_train-themes-light/app/assets/stylesheets/light/fields/super_select.css

```diff
@@ -91,6 +91,11 @@
     }
   }
 
+  .select2-container--default.select2-container--disabled .select2-selection--single,
+  .select2-container--default.select2-container--disabled .select2-selection--multiple{
+    @apply bg-slate-200 dark:bg-slate-700 hover:bg-slate-200 hover:dark:bg-slate-700
+  }
+
```

Comment:
This makes it so that disabled super selects have the same background colors as other disabled inputs. Without this disabling the address looks like this:

![CleanShot 2024-11-26 at 10 46 41](https://github.com/user-attachments/assets/a7460fbd-49bd-4e5e-b69e-c70957fc9013)